gnat2goto/driver/ureal_to_binary.adb

```diff
@@ -71,17 +71,11 @@ package body Ureal_To_Binary is
          --  writing to the first position
          Int_End := Int_Part'First + Integer (E);
 
-         if Int_End > Int_Part'Last then
-            raise Integer_Part_Too_Large;
-         end if;
-
-         pragma Assert (Int_End <= Int_Part'Last);
```

Comment:
I don't think these bits are necessary anymore with the changes to the buffer size, can you double check?

Reply:
Well we need to return `int_end` so the assignment above is necessary.

Reply:
No, I mean I don't think removing these parts is necessary

Reply:
I see, un-removed. Bellow as well.